### ISSUE TITLE
how can I take photo with bebop2?

### ISSUE THREAD

**Issue by EagleX1995** [amymcgovern/pyparrot#144]
bebopVision.open_video() won't return current images. All I get were the old images.

**Comment by amymcgovern** [comment]
Did you read the documentation and the sample code?  That is not how you get images.

https://pyparrot.readthedocs.io/en/latest/vision.html

**Comment by EagleX1995** [comment]
I had run the sample code demoBebopVision.py
The program returned the images from the previous execution.

**Comment by amymcgovern** [comment]
What OS are you on?  sounds like your ffmpeg program didn't clean out the old images.

**Comment by EagleX1995** [comment]
The latest macOS. So please tell me how can I clean out the old images. Thank you very much.

**Comment by amymcgovern** [comment]
that makes it easy!  It should be deleting them automatically though.  Since Mac OS is my primary environment, I'll go look into that.  In the meantime, find the images directory which should be inside your pyparrot installation and remove everything in there.

**Comment by EagleX1995** [comment]
Thank you. This is really helpful. Feels like removing all images under the directory after each execution can be the simplest solution for me. I was keeping rebooting and reseting my bebop LOL

**Comment by amymcgovern** [comment]
Yeah this is definitely in the directory and not on the bebop since you are using ffmpeg.  If you use the demoBebopVisionGUI version that won't happen (which is what I use 99% of the time).

**Comment by amymcgovern** [comment]
I fixed this today.  ffmpeg now cleans out the images directory each restart by default (but it can be turned off if you prefer that).

**Comment by amymcgovern** [comment]
Also fixed the bug where ffmpeg wasn't being killed properly!
